src/Microsoft.AspNet.Server.Kestrel/Http/Frame.Templated.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
I expect my heavy use of `ctrlr+,` is going to cause me to dislike `Frame.OfT.cs`. I'm already thoroughly annoyed that `Frame.FeatureCollection.cs` is the first result when I type "frame".

Do what you think is right <sup>but I might really dislike you if you make the wrong decision</sup>